utils/deploys/deployToken.ts

```diff
@@ -7,6 +7,8 @@ import { IndexToken__factory } from "../../typechain/factories/IndexToken__facto
 import { MerkleDistributor__factory } from "../../typechain/factories/MerkleDistributor__factory";
 import { Vesting__factory } from "../../typechain/factories/Vesting__factory";
 import { OtcEscrow__factory } from "../../typechain/factories/OtcEscrow__factory";
+import { FTCVesting__factory } from "@typechain/factories/FTCVesting__factory";
+import { FTCVesting } from "@typechain/FTCVesting";
```

Comment:
Can you export FTCVesting in `utils/contracts.ts` so that you can import it with the rest of the contract types on line 4